x-pack/legacy/plugins/lens/public/xy_visualization_plugin/types.ts

```diff
@@ -71,10 +71,15 @@ const axisConfig: { [key in keyof AxisConfig]: ArgumentType<AxisConfig[key]> } =
   },
 };
 
-export interface YConfig extends AxisConfig {
+export interface YState extends AxisConfig {
   accessors: string[];
 }
 
+export type YConfig = AxisConfig &
```

Comment:
Nice pattern, I think this will get very handy in the future as we pull more information out of the table data (e.g. formatting)